dev-ruby/hashicorp-checkpoint/hashicorp-checkpoint-0.1.5-r1.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+USE_RUBY="ruby25 ruby26 ruby27"
+
+RUBY_FAKEGEM_EXTRADOC="README.md"
+RUBY_FAKEGEM_GEMSPEC="ruby-checkpoint.gemspec"
```

Comment:
Missed this initially but there is no need to specify the gemspec file in this case. In this case we install from the gem itself and that already contains a copy we can use.

specifying the gemspec file is only needed when we install from e.g. a github tarball or when we need to tinker with it.

Reply:
Ah, because you include the gemspec we now get this output:
 * Running install phase for ruby25 ...
fatal: not a git repository (or any parent up to mount point /var/tmp)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).

Because the gemspec file tries to run git. This means that this ebuild won't install on a system without git as it is.

I'll fix this when merging this commit by removing the gemspec line.

Reply:
Ok thank you the review / merge.

I did missused the RUBY_FAKEGEM_GEMSPEC when it was not needed I will keep that in mind.